demo/tests/test_hello.py

```diff
@@ -1,7 +1,15 @@
-from ..hello import main
+import sys
 
+from ..hello import get_hello, main
 
-def test_main(capsys):
+
+def test_get_hello():
+    assert get_hello() == 'Hello World!'
+    assert get_hello('Kitty') == 'Hello Kitty!'
```

Comment:
You may want to consider this:

```suggestion
    assert get_hello() == 'Hello World!'
    assert get_hello('World') == 'Hello World!'
    assert get_hello('Matt') == 'Hello Matt!'
```